downstream/assemblies/platform/assembly-installing-hub-operator.adoc

```diff
@@ -15,6 +15,8 @@ ifdef::context[:parent-context: {context}]
 [role="_abstract"]
 You can use these instructions to install the {HubName} operator on {OCP}, specify custom resources, and deploy {PlatformNameShort} with an external database.
 
+{HubNameStart} configuration should be done through the custom resource specification and not directly to the Hub deployments because the changes will not persist. Hub settings should always be set as lowercase on the Hub custom resource specification.
```

Comment:
The second sentence is good as is.  

For the first bit, the point we want to get across is that the automationhub.spec.pulp_settings defined settings will take precedence over any settings defined in the Hub UI.  The operator will maintain the state specified by the parameters in automationhub.spec (CR spec).

Reply:
@rooftopcellist. This statement has been corrected as discussed for Hub, and a similar note has been added to the Controller section as well.

Reply:
Thanks! looks good.